### PR TITLE
Feat examples

### PR DESCRIPTION
**Pull request by nmerget**

### CODE REVIEW

File: examples/react-example/src/index.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@use "@db-ui/foundations/build/scss/rollup.assets-paths" as *;
+@use "@db-ui/foundations/build/scss/icon/icons" as *;
+@use "@db-ui/foundations/build/scss/db-ui-foundations" as *;
+@use "@db-ui/foundations/build/scss/variables.global" as *;
+
+@import "@db-ui/foundations/build/scss/color-classes";
```
Comment (mfranzke) [inline examples/react-example/src/index.scss:6]: Maybe a quick comment for each included file both here and within the README adds some more value for the consuming users on its contents and why those get included.

File: .xo-config.json [hunk 2]
```diff
@@ -14,6 +14,7 @@
 		"n/file-extension-in-import": 0,
 		"@typescript-eslint/no-unsafe-return": 0,
 		"@typescript-eslint/no-unsafe-assignment": 0,
-		"unicorn/prefer-top-level-await": 0
+		"unicorn/prefer-top-level-await": 0,
+		"unicorn/filename-case": 0
```
Comment (mfranzke) [inline .xo-config.json:18]: why do we need to define this rule (totally deactivating any naming style)?

File: examples/vue-example/src/main.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import { createApp } from 'vue';
+import '@db-ui/v-components/dist/style.css';
```
Comment (mfranzke) [inline examples/vue-example/src/main.ts:2]: I can't find this `import` on the related `examples/react-example/src/main.tsx` file for React – is this done on purpose ?

Comment (nmerget) [reply examples/vue-example/src/main.ts:2]: Yes, I think it is some vue-specific thing...

File: .xo-config.json (hunk 2 above)
Comment (nmerget) [reply .xo-config.json]: fixed

File: examples/react-example/src/index.scss (hunk 1 above)
Comment (nmerget) [reply examples/react-example/src/index.scss]: fixed

**Review by annsch — APPROVED** [review]

**Pull request merged**